src/offset/library.cairo

```diff
@@ -193,81 +183,124 @@ namespace CarbonableOffseter {
         }
         // [Check] Owned token id
         let (contract_address) = get_contract_address();
-        let (carbonable_project_address) = carbonable_project_address_.read();
+        let (carbonable_project_address) = CarbonableOffseter_carbonable_project_address_.read();
         // [Check] Throws error if unknown token id
         let (owner) = IERC721.ownerOf(
             contract_address=carbonable_project_address, tokenId=token_id
         );
-        with_attr error_message("CarbonableOffseter: token_id has not been registred") {
+        with_attr error_message("CarbonableOffseter: token_id has not been registered") {
```

Comment:
The purpose is to return the registered owner of the token, not the owner of the token (in case it has not been registered).
We don't want to replace `ownerOf` ERC-721 function.

It looks good as it is imo.